packages/react/src/components/Tooltip/TooltipHost.base.tsx

```diff
@@ -99,6 +99,8 @@ export class TooltipHostBase extends React.Component<ITooltipHostProps, ITooltip
         onMouseEnter={this._onTooltipMouseEnter}
         onMouseLeave={this._onTooltipMouseLeave}
         onKeyDown={this._onTooltipKeyDown}
+        role="none"
+        // WARNING: aria-describedby on this node provides no value, since it isn't allowed generic elements
```

Comment:
Do you know what the intent of this was, and whether there's a better way to achieve it? And regardless it might be good to update the docs for the `setAriaDescribedBy` prop.

Reply:
I think the intent was to provide a description, and the person who did it just wasn't aware that putting `aria-describedby` on a generic div doesn't actually work 😅.

I can definitely update the docs though! Should I mark it deprecated?

Reply:
Yeah I think that would make sense. Is there any other reasonable way to get the behavior described in the `setAriaDescribedBy` docs? If so you might add that too.

Reply:
Also, looks like the deprecation caused a lint error https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=213950&view=logs&j=258ec178-2d8b-5611-7b9b-60c5c95dae55&t=a347995d-6693-5610-51bd-74417b4bee80&l=3395

Reply:
I updated the comment for `setAriaDescribedby`, hopefully that'll help :). Lint error is resolved, and maybe someday I'll figure out a way to know which packages need `update-snapshots` without needing to just wait for the CI (or a root update-snapshots, which is somehow worse) 😭

Reply:
I wouldn't have realized charting and experiments would need snapshot updates in this case either 🤷‍♀️